src/var_reference/GaussianReference.jl

```diff
@@ -0,0 +1,44 @@
+"""
+A Gaussian mean-field variational reference (i.e., with a diagonal covariance matrix).
+"""
+@kwdef mutable struct GaussianReference <: VarReference
+    μ::Dict{Symbol, Any} = Dict{Symbol, Any}() # means
+    σ::Dict{Symbol, Any} = Dict{Symbol, Any}() # standard deviations
+    
+    function GaussianReference(μ, σ)
+        @assert length(μ) == length(σ)
+        new(μ, σ)
+    end
+end
+
+dim(var_reference::GaussianReference) = length(var_reference.μ)
+activate_var_reference(::GaussianReference, iterators::Iterators) = iterators.round ≥ 6 ? true : false
```

Comment:
Ideally this would be a function of the number of parameters in the model. I don't know what the best value would be so I just set it similar to the value from the VPT paper. Any ideas?

Reply:
Added entry to GaussianReference that stores the "6" as a default value but can be specified by the user.